### DISCUSSION TITLE
Chrome OS

### DISCUSSION THREAD

**Discussion by SkyJayne** [Flam3rboy/discord-bot-client#225] [Q&A]
I'm on a device that runs on Chrome OS and I was wondering if I could still use this. (sorry I'm stupid)

**Comment by r4h33q** [comment]
Hey @SkyJayne,
The answer to your question is yes. Here are the steps:
1. Go to settings then in the bottom there is "Linux(Beta)"
2. Press Turn On
3. After a lot of waiting the app "Terminal" will open by it self
4. Install [Linux .DEB version](https://github.com/Flam3rboy/discord-bot-client/releases/download/3.1.0/discord-bot-client_3.1.0_amd64.deb)
5. Afterwards go to files and move the file "discord-bot-client_3.1.0_amd64.deb" from "Downloads" to "Linux Files" (THIS  IS VERY IMPORTANT)
6. Double click the file "discord-bot-client_3.1.0_amd64.deb" in Linux Files and press Install
7. After installation, go to Launcher or the button used to open your app library
8. Find the folder Linux Apps and open the app "Discord Bot Client"

Hope this helps! 😉
If not tell me.
<div align="right">-Lightner</div>
